package.json

```diff
@@ -34,9 +35,13 @@
     "eslint": "^2.13.1",
     "eslint-config-rackt": "^1.1.1",
     "jest": "^13.0.0",
+    "lodash.flow": "^3.5.0",
+    "lodash.map": "^4.6.0",
+    "lodash.sortby": "^4.7.0",
```

Comment:
I definitely see the convenience of having one import for development instead of multiple. I'll change it in a future PR.